src/sonic-yang-models/tests/yang_model_tests/tests_config/syslog.json

```diff
@@ -0,0 +1,49 @@
+{
+    "SYSLOG_SERVER_TEST": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13"
+                    },
+                    {
+                        "server_address": "20.1.1.1"
+                    }
+                ]
+            }
+        }
+    },
+    "SYSLOG_SERVER_1_NEG_TEST": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": ""
+                    }
+                ]
+            }
+        }
+    },
+    "SYSLOG_SERVER_2_NEG_TEST": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": "2001:aa:aa:aa"
+                    }
+                ]
+            }
+        }
+    },
+    "SYSLOG_SERVER_INVALID_IPADDR_TEST" : {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": "1111:22:33.1"
```

Comment:
Hi arlakshm, could you change it to an invalid IPv4 address? Like `1111.22.33.1`
From my local test, invalid IPv4 address will not be blocked.

Reply:
From my [Syslog server test#4725](https://github.com/Azure/sonic-mgmt/pull/4725), the invalid IPv4(test_syslog_server_tc3_xfail) was expected to fail but succeed.
```
def test_syslog_server_tc3_xfail(duthost, setup_env, op,
        dummy_syslog_server_v4, dummy_syslog_server_v6):
    """ Test expect fail testcase
    ("add", "10.0.0.587", "cc98:2008::1"), ADD Invalid IPv4 address
```

Reply:
address in the latest commit